project_index.php

```diff
@@ -0,0 +1,27 @@
+<?php
+require_once('functions.php');
+$db = new PDO('mysql:host=db; dbname=collection_project', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `charname`, `class`, `level`, `strength`, `dexterity`, `constitution`, `intelligence`, `wisdom`, `charisma` FROM `characters`;");
+// $query->bindParam(':email', $email);
+$query->execute();
+$characters = $query->fetchAll();
```

Comment:
I would suggest popping this step into a function as well, so any time you need the list of characters you can just call the function without having to copy and paste this query

Reply:
Note that because this function will touch the database, it cannot be unit tested